docs/guide/tutorial-3-appearance.md

```diff
@@ -70,7 +70,7 @@ The aim of this subsection is to apply a locally available texture to the ball.
 A texture on a rolling object can help to appreciate its movement.
 
 > **Hands-on #4**: add a locally available texture to the ball.
-1. Download the texture available [here](https://raw.githubusercontent.com/cyberbotics/webots/R2021b/projects/default/worlds/textures/red_brick_wall.jpg) and save it to your disk.
+1. Download the texture available [here](https://raw.githubusercontent.com/cyberbotics/webots/{{ webots.version.full }}/projects/default/worlds/textures/red_brick_wall.jpg) and save it to your disk.
```

Comment:
I have just checked again and it might be even better to add a new variable (and use it here), e.g. `url.github_raw` as an alternative to `url.github_tree`
https://github.com/cyberbotics/webots/blob/5389e9d01de62f65ef7ed30f4759c375882663fb/docs/js/showdown-extensions.js#L45

That way, if somebody changes the file in a custom branch it will still work.